image/verify/src/verifier.rs

```diff
@@ -749,7 +750,7 @@ impl<Env: ImageVerificationEnv> ImageVerifier<Env> {
 
         let log_info: ImageSvnLogInfo = ImageSvnLogInfo {
             manifest_svn: verify_info.svn,
-            manifest_min_svn: verify_info.min_svn,
+            reserved: 0,
```

Comment:
This is probably causing your ROM diff...

Reply:
Ah, fixed.